src/jloptions.c

```diff
@@ -194,7 +195,8 @@ static const char opts_hidden[]  =
     " --output-ji <name>       Generate a system image data file (.ji)\n"
     " --strip-metadata         Remove docstrings and source location info from system image\n"
     " --strip-ir               Remove IR (intermediate representation) of compiled functions\n\n"
-
+    " --emit-sanitizer={none*|asan|msan|tsan}\n"
```

Comment:
Name this option `-fsanitize` to match clang and gcc?

Reply:
I was wondering about this, that flag doesn't turn the sanitizers on per say, but only adds the function attribute. It doesn't run the passes that actually add the functions. So it might be confusing.

Reply:
Hmm, that is a fair point. Leave it for now, I suppose

Reply:
Given that we don't use the `-f$name` pattern, calling it `-fsanitize` seems weird. Also, we can't match clang and gcc exactly since we require two dashes for long options.